### ISSUE TITLE
Automate !help to read from a inherited documentation variable

### ISSUE THREAD

**Issue by InhoStudios** [VikingsDev/Dentaku#21]
Create some file that holds all the commands and their functions, and have the !help code read and parse that file to create the message response

**Comment by InhoStudios** [comment]
also perhaps get more information about the usage of a certain command by typing !help followed by the command name

**Comment by FrankWhoee** [comment]
Added a self.documentation variable to each class, and also a define_documentation method

**Comment by FrankWhoee** [comment]
In help.py you can use the method I used in main.py to import and run these functions:

```
module = __import__(command)
new_command = getattr(module, command)
instance = new_command(parameters, client=self)
```

**Comment by FrankWhoee** [comment]
Method of implementation would be to go through each python file in the Dentaku directory, and import each, and run define_documentation on each instance, access the documentation variable and use that information to generate a help function

**Comment by FrankWhoee** [comment]
20 points to the person who can complete this! If you resolve this issue, reference it in your PR by saying Resolved #21

**Comment by ratanawang** [comment]
what the heck that's so many points

**Comment by ratanawang** [comment]
can i claim this ;-;

**Comment by FrankWhoee** [comment]
> can i claim this ;-;

Assigned!

**Comment by ratanawang** [comment]
> Added a self.documentation variable to each class, and also a define_documentation method

You want me to do this? Or have you already done it?

**Comment by ratanawang** [comment]
nvm i'll figure it out

**Comment by FrankWhoee** [comment]
Resolved by #37
